frontend/apps/filemanager/filemanagershortcuts.lua

```diff
@@ -6,21 +6,22 @@ local InputDialog = require("ui/widget/inputdialog")
 local Menu = require("ui/widget/menu")
 local Screen = require("device").screen
 local UIManager = require("ui/uimanager")
+local lfs = require("libs/libkoreader-lfs")
```

Comment:
Unneeded?

Reply:
The script *does* work without this inclusion. But then lfs is loaded from a global place. While in most classes lfs is loaded as a local library. So it seems more consistent to do that here also.